libs/sdk-ui-kit/src/Dialog/ShareDialog/ShareDialogBase/ShareGranteeBase.tsx

```diff
@@ -0,0 +1,42 @@
+// (C) 2021 GoodData Corporation
+import React, { useMemo } from "react";
+import { useIntl } from "react-intl";
+import { ConfirmDialogBase } from "../../ConfirmDialogBase";
+import { ContentDivider } from "./ContentDivider";
+import { ShareGranteeContent } from "./ShareGranteeContent";
+import { IShareGranteeBaseProps } from "./types";
+import { sortGranteesByName } from "./utils";
+
+/**
+ * @internal
+ */
+export const ShareGranteeBase: React.FC<IShareGranteeBaseProps> = (props) => {
+    const { grantees, owner, isDirty, onCancel, onSubmit, onGranteeDelete, onAddGranteeButtonClick } = props;
+    const intl = useIntl();
+
+    const granteeList = useMemo(() => {
+        const granteeSorterByName = sortGranteesByName(intl);
```

Comment:
Minor: You could just use the method directly in `sort` and save a line :)

Reply:
Done